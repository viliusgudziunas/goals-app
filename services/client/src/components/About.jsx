import React from 'react';

const About = () => {
  return (
    <div className='container'>
      <br />
      <h1>About</h1>
      <hr />
      <p>Add something relevant here</p>
    </div>
  );
};

export default About;
