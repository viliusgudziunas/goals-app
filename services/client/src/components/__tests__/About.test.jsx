import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import About from '../About';

describe('About', () => {
  it('renders a snapshot properly', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  const wrapper = shallow(<About />);
  it('renders properly', () => {
    const header = wrapper.find('h1');
    expect(header.text()).toBe('About');

    const paragraph = wrapper.find('p');
    expect(paragraph.text()).toBe('Add something relevant here');
  });
});
