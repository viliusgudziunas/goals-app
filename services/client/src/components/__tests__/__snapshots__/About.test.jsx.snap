// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should match the snapshot 1`] = `"<div class=\\"container\\"><br/><h1>About</h1><hr/><p>Add something relevant here</p></div>"`;
